config/image/oss/kustomization.yaml

```diff
@@ -7,4 +7,4 @@ images:
   newTag: '3.3'
 - name: kic-placeholder
   newName: kong/kubernetes-ingress-controller
-  newTag: '2.10.2'
+  newTag: '2.10'
```

Comment:
Why not `2.10.3`?

Reply:
Just to not be forced to bump it every time we release a patch. AFAIK we always want to use the latest minor patch. That's also how we specify the gateway image.